script.js

```diff
@@ -1,14 +1,29 @@
 function shifumi() {
-    const user_choice = ['rock', 'paper', 'scissors']
-    const choice = prompt(`Choose one of the following: ${user_choice.join(', ')}.`)
+    const user_choice = ['rock', 'paper', 'scissors'];
+    const choice = prompt(`Choose one of the following: ${user_choice.join(', ')}.`);
 
     if (!user_choice.includes(choice)) {
         alert(`Attention, ${choice} is not one of the ${user_choice.length} choices`);
         return shifumi()
     }
 
     alert(`Congratulations, you chose ${choice}`);
-}
 
+    //AI choice
+    const ia = Math.floor(Math.random() * user_choice.length)
+    const ia_choice = user_choice[ia]
+    alert(`AI chose ${ia_choice}, let's play !`);
+
+    //who wins
+    if (ia_choice.includes(choice)) {
```

Comment:
since you are not looking for a substring of the IA choice, you can directly use a pure equality here instead (`===`)